example/windows/win32_window.cc

```diff
@@ -100,6 +112,18 @@ Win32Window::MessageHandler(HWND hwnd, UINT const message, WPARAM const wparam,
       Destroy();
       return 0;
 
+    case WM_DPICHANGED: {
+      // Resize the window only for toplevel windows which have a suggested
+      // size.
+      auto lprcNewScale = reinterpret_cast<RECT *>(lparam);
```

Comment:
Use style-compliant variable names, not Windows-style naming, per style guide.

Reply:
Done